src/clr-angular/data/datagrid/datagrid.ts

```diff
@@ -143,6 +143,33 @@ export class ClrDatagrid<T = any> implements AfterContentInit, AfterViewInit, On
 
   @Output('clrDgSingleSelectedChange') singleSelectedChanged = new EventEmitter<T>(false);
 
+  private _clrDetailSingleSelection: string = '';
+  @Input('clrDetailSingleSelection')
+  get clrDetailSingleSelection(): string {
+    return this._clrDetailSingleSelection || this.commonStrings.selectRow;
+  }
+  set clrDetailSingleSelection(value: string) {
+    this._clrDetailSingleSelection = value;
+  }
+
+  private _clrDetailActionableSelection: string = '';
+  @Input('clrDetailActionableSelection')
+  get clrDetailActionableSelection(): string {
+    return this._clrDetailActionableSelection || this.commonStrings.selectRow;
+  }
+  set clrDetailActionableSelection(value: string) {
+    this._clrDetailActionableSelection = value;
+  }
+
+  private _clrDetailExpandableRow: string = '';
+  @Input('clrDetailExpandableRow')
```

Comment:
```@Input() clrDgExpandableAriaLabel = this.commonStrings.datagridExpandableOverflow;```

This would be used for both expandable rows and master/detail.